src/preferences/dialog/dlgprefinterface.cpp

```diff
@@ -181,6 +183,31 @@ DlgPrefInterface::DlgPrefInterface(
     int inhibitsettings = static_cast<int>(m_pScreensaverManager->status());
     comboBoxScreensaver->setCurrentIndex(comboBoxScreensaver->findData(inhibitsettings));
 
+    // Multi-Sampling
+#ifdef MIXXX_USE_QML
+    if (CmdlineArgs::Instance().isQml()) {
+        mulitSamplingComboBox->clear();
+        mulitSamplingComboBox->addItem(tr("Disabled"), 0);
+        mulitSamplingComboBox->addItem(tr("2x MSAA"), 2);
+        mulitSamplingComboBox->addItem(tr("4x MSAA"), 4);
+        mulitSamplingComboBox->addItem(tr("8x MSAA"), 8);
+        mulitSamplingComboBox->addItem(tr("16x MSAA"), 16);
+
+        m_multiSampling = m_pConfig->getValue(ConfigKey(kPreferencesGroup, kMultiSamplingKey), 0);
```

Comment:
actually thinking about it, shouldn't the default value be 4 instead? otherwise we're back to the AA-less default, resulting in poor out-of-the-box experience. wdyt?

Reply:
Yep, that would makes sense as default, but I'm wondering what would be the impact on lightweight setup. I've updated it anyway.

Reply:
I think those are niche enough. On lightweight setups we can reduce it down (maybe autodetect the framerate and adjust defaults downward accordingly?) I'd prefer the OOTB experience to be more polished for the majority of users.